velox/exec/RowContainer.h

```diff
@@ -525,6 +525,10 @@ class RowContainer {
         stringAllocator_.freeSpace());
   }
 
+  // Returns the average size of rows in bytes stored in this container.
+  // Returns -1 if the container is empty.
+  int64_t estimateRowSize() const;
```

Comment:
Should we return std::optional instead of a special value -1?

Reply:
Revised.